api/src/main/java/org/openmrs/module/ugandaemr/metadata/core/EncounterTypes.java

```diff
@@ -324,4 +324,19 @@ public String uuid() {
             return "afcdfcd8-defe-11eb-ba80-0242ac130004";
         }
     };
+    public static EncounterTypeDescriptor WALKIN_CLIENT_CAPTURE = new EncounterTypeDescriptor() {
+        @Override
+        public String name() {
+            return "Walkin Art Service";
```

Comment:
@semujju Change this to "Emergency ART Service Encounter"

Reply:
Done